Programs/2/cribbage.py

```diff
@@ -0,0 +1,58 @@
+def score_3_of_a_kind(cvals):
+    if cvals[0] == cvals[1] and cvals[0] == cvals[2]: # 3 of a kind
+        return 6
+    return 0
+
+def score_pair(cvals):
+    if cvals[0] == cvals[1] or cvals[1] == cvals[2]: # pair
+        return 2
+    return 0
+
+def score_sequence(cvals):
```

Comment:
Doesn't account for out of order sequence, since that is allowed. Better solution could be
def consecutive(vals):
    the_sum = vals[0] + vals[1] + vals[2]
    x = (the_sum - 3) / 3
    if (x in vals) and (x + 1 in vals) and (x + 2) in vals:
        return 3
    return 0

Reply:
[Line 32](https://github.com/blackbelt238/CSCI-127/pull/5/files#diff-78dffc9fe68dee545dceb2017752f890R32) indeed accounts for this.